src/generators.rs

```diff
@@ -856,10 +881,15 @@ fn gen_tuple_get(scale: u16, input_size: u16) -> (Option<String>, String) {
         ));
     }
     println!("{}", tuple);
-    (None, format!("(let ((test-tuple {})) {})", tuple, body))
+
+    GenOutput::new(
+        None,
+        format!("(let ((test-tuple {})) {})", tuple, body),
+        input_size,
```

Comment:
So here, the "unit" of TupleGet is the length of the TupleData - is the intention to actually calculate that value and return it here? 

^ similar comment for multiple of the functions in this file

Reply:
yes, that's the intention, but will be implemented in a followup PR